README.md

```diff
@@ -25,12 +25,18 @@ idea of where things are, and where they are going:
 We haven't created issues for the other sections yet, but they will be coming
 soon. For dev-related work please see [General - Updates & Fixes][general-url].
 
-The alpha version of webpack 5 is out and we are lagging behind with documenting all of the related changes. See [this refined search][webpack5-milestone-url] for the list of relevant documentation requests.
-
 ## Translation
 
 To help translate this documentation please jump to the [translation branch][translate-url].
 
+## Versioning
+
+Since webpack 4 we have created a subdomain-based archive for older states of documentation
+matching older webpack version. webpack 4's documentation is available at
+https://v4.webpack.js.org/ and is deployed from https://github.com/webpack/v4.webpack.js.org/tree/gh-pages
```

Comment:
can we add it as a link ,so that user don't have to copy paste and instead can directly go the pages
`https://v4.webpack.js.org/` -> [https://v4.webpack.js.org/](https://v4.webpack.js.org/)